ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1022,7 +1022,7 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
             <OldCdrVersionModal
                 onCancel={() => {
                   this.setState(fp.set(['workspace', 'cdrVersionId'],
-                    getDefaultCdrVersionForTier(this.state.workspace, cdrVersionTiersResponse)));
+                    getDefaultCdrVersionForTier(this.state.workspace, cdrVersionTiersResponse).cdrVersionId));
```

Comment:
The fix!  It was trying to set `workspace.cdrVersionId` to the whole CDR Version structure instead of the ID